views/templates/admin/_configure/helpers/form/form.tpl

```diff
@@ -29,7 +29,7 @@
 		<div class="col-lg-9">
 			{foreach from=$languages item=language}
 				{if $languages|count > 1}
-					<div class="translatable-field lang-{$language.id_lang}" {if $language.id_lang != $defaultFormLanguage}style="display:none"{/if}>
+					<div class="translatable-field lang-{$language.id_lang}" {if $language.id_lang != $defaultFormLanguage} style="display:none"{/if}>
```

Comment:
Hi,

Extra space is not required as you have already one before {if $language.id_lang.

Best regards